stdlib/test/collections/test_list.mojo

```diff
@@ -768,3 +990,7 @@ def main():
     test_list_count()
     test_list_add()
     test_list_mult()
```

Comment:
**Suggestion** Since the pattern of creating a parameterized "inner" test function repeats for essentially all of the test functions in this file, I suggest we instead parameterize the top-level test functions (i.e. don't have any inline functions at all), and then additionally move the calls from the `main()` driver function out to their own function, so that we have:

```
def run_list_tests[sbo_size: Int]():
    test_mojo_issue_698[sbo_size]()
    test_list[sbo_size]()
    test_list_clear[sbo_size]()
     ...

def main():
    unroll[run_list_tests, 10]()

```

Reply:
I indeed though about this, and I was wondering how compatible it will be with the test runner currently in the works. Do you have any info about this? As far as I know the test runner doesn't execute `main()`